bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/twinguard/BoschTwinguardHandler.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
Since this is a new year, please update the copyright date for all your files.
```suggestion
 * Copyright (c) 2010-2021 Contributors to the openHAB project
```

Reply:
Hello @coeing I will taking care of the copyright update and move the final fields.

Reply:
A rebase on main branch was necessary to get the latest maven file header format.